CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing with NovusCore
+
+Awesome! You want to help and contribute to NovusCore!
+
+You can contribute in different ways for NovusCore, either by opening bug reports, suggesting fixes for already open issues, proposing fixes 
+with Pull Requests (PR) or by suggesting new ideas.
+
+## What should I know before I start?
+
+1. First off, NovusCore has 3 projects which you can contribute to:
+
+Our [World of Warcraft 3.3.5 Branch](https://github.com/novuscore/NovusCore)
+
+Our [NovusCore-GMClient](https://github.com/novuscore/NovusCore-GMClient)
+
+Our [NovusCore-Client](https://github.com/novuscore/NovusCore-Client)
+
+Make sure you choose the correct project to contribute to.
+
+2. Do not open an issue or PR to ask a question, you will get a quicker response in our [Discord Channels](https://discordapp.com/invite/W8XTFJ?utm_source=Discord%20Widget&utm_medium=Connect).
+
+## Opening a Bug Report 📝
+
+1. First thing you should do is to make a quick search to see if the same or a similar issue is already opened.
```

Comment:
1. Start by making sure the bug you are reporting hasn't already been reported, you can do this by searching through the issue list. In the event that there already is an open issue, please contribute your information to that issue as it will be easier for the developers to have all the information in one place.